src/Compilers/CSharp/Portable/Binder/Binder_Lookup.cs

```diff
@@ -1391,7 +1391,11 @@ internal SingleLookupResult CheckViability(Symbol symbol, int arity, LookupOptio
                 ? ((AliasSymbol)symbol).GetAliasTarget(basesBeingResolved)
                 : symbol;
 
-            if (!IsInScopeOfAssociatedSyntaxTree(unwrappedSymbol))
+            if ((options & LookupOptions.MustNotBeParameter) != 0 && unwrappedSymbol is ParameterSymbol)
```

Comment:
>(options & LookupOptions.MustNotBeParameter) != 0 && unwrappedSymbol is ParameterSymbol [](http://example.com/codeflow?start=16&length=87)

Shouldn't this be considered also in `CanAddLookupSymbolInfo`?

Reply:
> Shouldn't this be considered also in CanAddLookupSymbolInfo?

I don't think there is a scenario where that would make a difference. This isn't an option that is expected to be used generally